sfcasts/dashboard.md

```diff
@@ -1,95 +1,61 @@
+
+Okay, so there's a few things to notice here. First is that we do have a `/admin` route. This isn't fancy, and there's really nothing "EasyAdmin" about it. This is just how we create routes in Symfony. This is a PHP 8 attribute route, which you may or may not be familiar with. I've typically used annotations until now, but now that I'm using PHP 8, I'll be using attributes instead of annotations. Don't worry, though! They work exactly the same. If you're still using PHP 7, then you can use annotations just fine.
+
+The `DashboardController` is just a normal controller, but it extends `AbstractDashboardController`. I'll hold "Cmd" and open that. 
+
+This implements `DashboardControllerInterface`. So this is a controller, but by implementing the interface, it gives it special superpowers. There are a number of methods inside of here that we can override to configure what our dashboard looks like. We'll be doing that little by little throughout this tutorial. 
+
+Because this is just a normal route in controller, there's really nothing special about it. It's not automatically secured and there's no security being applied to this. I mean, check it out: I'm not even logged in right now, and I am successfully on my admin dashboard.
+
+So let's secure it! I'll also do this with annotations. I already have SensioFrameworkExtraBundle installed. So I can say `#[IsGranted()]` and hit "tab" to auto complete that. Now we'll look for `ROLE_ADMIN`. That's kind of my base admin role, so all admin users will have that role. Now when I refresh... Beautiful! We bounced back over to the login page!
+
+So to log in, if you open `src/DataFixtures/AppFixtures`, I have a bunch of dummy users in the database. There is a super admin, a normal admin, and then somebody known as a moderator. We'll talk more about those later, when we get deeper into how to secure different parts of your admin with different roles.
+
+I'll log in with "admin@example.com"... password "admin password"...
+
+And... beautiful! Now we're back to our dashboard!
+
+Of course, if you want to, instead of securing `IsGranted` with the PHP attribute, you can use `$this->denyAccessUnlessGranted()`. You can also go to `config/packages/security.yaml` and, down at the bottom, you can always just use access control for this - whichever one you like better.
+
+So, our dashboard is the jumping off point for our admin, but there's nothing too special here. This page has a title, some menu items, and a nice little user menu over here. Eventually, we'll be able to actually generate something cool on this page, like links or graphs or something. By the way, styling is all done in Bootstrap 5 with FontAwesome.
+
+So let's see if we can customize the dashboard a little bit. One of the best things about EasyAdmin is that all the config is done in PHP, usually via methods in your controller. So, for example, want to configure the dashboard? There is a configured dashboard method for that!
```

Comment:
These 2 paragraphs are a good example of me over-using "So"... but I think that it doesn't sound as good without them. So +1 for keeping them.

Reply:
There is -> "There's"

"`configureDashboard() method" (technical term that you'll see in the video)